### ISSUE TITLE
Universal Temp

### ISSUE THREAD

**Issue by potatoqualitee** [PowershellFrameworkCollective/psframework#381]
$env:temp does not work on Linux and the resulting discussion gets heated at: https://github.com/PowerShell/PowerShell/issues/4216 
 
Can you wrap  [system.io.path]::GetTempPath() as `Get-PsfTempPath`? 

I'd like a universal temp path without remembering the syntax plus $env:temp doesn't work well on windows since it's DOS-shaped.

/cc @luzkenin

**Comment by FriedrichWeinmann** [comment]
Sounds like something I'd want to do :)

Specifically: A command that handles all the well-known paths.
Handling OS specific options, backed by the configuration system (so users can move temp if it doesn't work for them or they just want it elsewhere)

**Comment by potatoqualitee** [comment]
awesome! that would be insanely useful since psframework is part of all of my modules 💃

**Comment by FriedrichWeinmann** [comment]
Implemented and released in 1.1.59

```powershell
Get-PSFPath
Set-PSFPath
```

Supported default paths: Temp, AppData, LocalAppdata, ProgramData

Backed by configuration:

```
PSFramework.Path.AppData
PSFramework.Path.LocalAppData
PSFramework.Path.ProgramData
PSFramework.Path.Temp
```

Freely extensible by additional named paths (Both Name and Path are arbitrary)
